services/basic_types.proto

```diff
@@ -1415,3 +1415,20 @@ message TokenAssociation {
     TokenID token_id = 1;  // The token involved in the association
     AccountID account_id = 2; // The account involved in the association
 }
+
+/**
+ * Related to EIP-2929 and EIP-2930 addresses can be provided for "warm up" in EVM before the execution of the
+ * transaction in order to save gas.
+ */
+message ContractAccessEntry {
+
+  /**
+   * The address of the smart contract that should be "warmed up"/loaded in the EVM
+   */
+  ContractID contract_ID = 1;
+
+  /**
+   * The storage keys of the smart contract that should be "warmed up"/loaded in the EVM
+   */
+  repeated bytes storage_keys = 2;
```

Comment:
What is the format of the keys on Hedera? Can the `Key` object be used here? If not, should document the format expected.

Reply:
32 bytes or less.  Strictly speaking it is a UInt256 but for space it is right packed and zero filled from the left (not sign extended). There is no fixed arrays or 256 byte values in protobuf or we would have used those.

Reply:
Ok, would be good to have the protobuf doc reflect something along the lines of your comment.